### PR TITLE
update for `data-attributes` on `mr-app`

### PR DESCRIPTION
**Pull request by hanbollar**
related to https://github.com/Volumetrics-io/mrjs/pull/660#pullrequestreview-2059117187

### CODE REVIEW

File: source/attributes/data-lighting.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# data-lighting
+
+```html
+<mr-app data-lighting="enabled=true; shadows:true; color:#faa; intensity:0.2; radius:4;"> ... </mr-app>
```
Comment (lobau) [inline source/attributes/data-lighting.md:4]: this is a bit weird: `enabled=true; shadows:true;`
Is that a mistake or an unfortunate code thingie we have to deal with?
`enabled:true; shadows:true;` would be nicer

File: source/attributes/data-occlusion.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+```html
+<mr-app data-occlusion="..????"> ... </mr-app>
+```
+
+Occlusion setup can be attached to an [`mr-app`](/doc/mr-app) using `data-occlusion`:
+
+### `data-occlusion`
+
+... not sure what to put here but it seems to do with the anchor system?
```
Comment (lobau) [inline source/attributes/data-occlusion.md:11]: LOL. I was gonna say, I don't know what data-occlusion is, and was gonna ask for more details.

**Review by lobau — COMMENTED** [review]
Left a few comments. Otherwise it looks good!

File: source/attributes/data-occlusion.md (hunk 2 above)
Comment (hanbollar) [reply source/attributes/data-occlusion.md:11]: XD def something i'm passing to @michaelthatsit atm from legacy setup

File: source/attributes/data-lighting.md (hunk 1 above)
Comment (hanbollar) [reply source/attributes/data-lighting.md:4]: good catch - thankfully i think that's just a bad typo on my part - i'll clean that up ~

**Comment by hanbollar** [comment]
going to merge this in since it's a blocker for the main `mrjs` pr - but still leaving the question open ended for @michaelthatsit to loop back on for `data-occlusion`

**Pull request merged**